google/cloud/compute/accelerator_types/v1/accelerator_types.proto

```diff
@@ -12,6 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Generated by the C++ microgenerator.
+// DO NOT EDIT. If you make any local changes, they will be lost.
```

Comment:
The magic `DO NOT EDIT` string will prevent formatting with `clang-format`.  Intentional?